docs/release-notes/dont-send-docker-creds-to-any-server.txt

```diff
@@ -0,0 +1,11 @@
+:orphan:
+
+**Improvements**
+
+-  Security: Setting ``registry_auth.serveraddress`` will now only send credentials to the server
+   configured. Not setting ``registry_auth.serveraddress`` is now deprecated and will be required
+   soon.
+
+-  Agent: Users may now run ``docker login`` on agent host machines to authenticate with Docker
+   registries. Note if an agent is running inside a Docker container then ``~/.docker/config.json``
+   will need to be mounted to the same path inside the container.
```

Comment:
Actually it needs to be mounted to `/root/.docker/config.json` since that will be `$HOME/.docker/config.json` inside the container (which by default runs as root).